contracts/deploy/foreign/zksync.js

```
@@ -23,9 +23,11 @@ async function deployForeignProxy({ deploy, from, parameters, homeNetworkName, h
   const surplus = eth("0.03"); // The surplus will be automatically reimbursed when the dispute is created.
   const l2GasLimit = 1500000; // Gas limit for a tx on L2.
   const l2GasPerPubdataByteLimit = 800;
+  const chainId = await getChainId();
```

ESLint

[error] 'getChainId' is not defined.

(no-undef)